launch/docker_test.go

```diff
@@ -496,16 +496,12 @@ func TestHelperProcess(t *testing.T) {
 		}
 		os.Exit(0)
 	case "SUCCESS_TO_KILL":
-		if subcmd == "command" {
+		if subcmd == "command" || subcmd == "sleep" {
```

Comment:
How about unifying `command` and `sleep`?

Reply:
thx.
I fixed.